docs/references/expo/use-sso.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: '`useSSO()`'
+description: Clerk's useSSO() hook is used to create a new SSO flow.
+---
+
+The `useSSO()` hook is used to create a new SSO flow. It can be used in both web and native apps.
+
+## Parameters
+
+<Properties>
+  - `strategy`
+
+  - [`OAuthStrategy`](/docs/references/javascript/types/sso#o-auth-strategy)
+
+  - [`EnterpriseSSOStrategy`](/docs/references/javascript/types/sso#enterprise-sso-strategy)
+
+  - `strategy?`
+
+    - `string`
+
+    Select the SSO strategy. The `enterprise_sso` value depends on the object's `identifier` value. Possible `strategy` values are:
+
+    - `oauth_<provider>`: The user will be authenticated with their social sign-in account. [See available social providers](/docs/references/javascript/types/oauth#o-auth-provider).
+    - `enterprise_sso`: The user will be authenticated either through SAML or OIDC, depending on the configuration of the [enterprise connection](/docs/authentication/enterprise-connections/overview) matching the identifier.
+
+    ---
+
+    - `redirectUrl?`
+    - `string`
+
+    The URL or path to redirect to after the SSO flow is complete.
+
+    ---
+
+    - `unsafeMetadata?`
+    - [`SignUpUnsafeMetadata`](/docs/references/javascript/types/metadata#sign-up-unsafe-metadata)
+
+    Unsafe metadata to be passed to the SSO provider.
+</Properties>
+
+## Returns
+
+The `useSSO()` hook returns the `useSSOFlow()` method, which you can use to initiate the SSO flow.
+
+The `startSSOFlow()` method has the following signature:
+
+```ts
+const startSSOFlow: (startSSOFlowParams?: StartSSOFlowParams) => Promise<StartSSOFlowReturnType>
+```
```

Comment:
It says the hook returns `useSSOFlow` but the function below is named `startSSOFlow`. I can't find either in the `dashboard` or `javascript` repo.

Reply:
`useSSO` returns `startSSOFlow` -> https://github.com/clerk/javascript/pull/4880

This feature is not currently merged therefore you cannot find it in `clerk/javascript` repo